lua/core/clock.lua

```diff
@@ -13,10 +19,15 @@ end
 local SLEEP = 0
 local SYNC = 1
 
+--- pause coroutine execution for s seconds.
+-- @tparam integer s : seconds
```

Comment:
missing requirement that it should be called from a function passed to clock.run

Reply:
seconds parameter isn't integer, is float

Reply:
i suggest using word 'suspend' instead of 'pause', that matches lua terminology better, see https://www.lua.org/pil/9.1.html